src/main/java/Duke.java

```diff
@@ -1,10 +1,138 @@
+import java.util.Objects;
+import java.util.Scanner;
 public class Duke {
+
+    static final int MAX_TASKS = 100;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        Scanner reader = new Scanner(System.in);
+        Task[] tasks = new Task[MAX_TASKS];
+        int counter = 0;
+        System.out.println("Hi, I'm bob");
+        System.out.println("What's up");
+        String word = reader.nextLine();
+
+        while (!word.equals("bye")) {
+
+            if (word.equals("list")) {
+                if (counter == 0) {
+                    System.out.println("No tasks!");
+                } else {
+                    getList(tasks, counter);
+                }
+                word = reader.nextLine();
+
+            //mark, unmark, or add
+            } else {
```

Comment:
Perhaps you can avoid the `else` statement here to make the happy path more prominent?

https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#make-the-happy-path-prominent

```suggestion
            continue;
            //mark, unmark, or add
```